### ISSUE TITLE
\n\t causes odd terminal output on Windows

### ISSUE THREAD

**Issue by ctwadv** [docker/for-win#1112]
### Expected behavior

Anything which prints "\n\t" would print always print newline, followed by a tab, and that newline would be appropriately printed.

### Actual behavior

A newline followed by a tab, printed by an in-container process, often exhibits CR-without-LF behavior, resetting the cursor to the leftmost column.  In some cases, a single \n\t can sometimes overwrite multiple lines of terminal output - see "1       4444" below.

### Information

Something about the "\n\t" sequence causes the Windows terminal to mangle output and begin overwriting lines.  Below is an example log of a brand new 'cmd' session.  The first two runs come out fine.  The third run gets mangled, because the output reaches the bottom of the 25-row window.  The fourth run is mangled differently, because the cursor started at the bottom of the window.  Fifth and subsequent runs of this same command produce output identical to the fourth.

```Microsoft Windows [Version 10.0.10586]
(c) 2015 Microsoft Corporation. All rights reserved.

C:\Users\ctw>docker run --rm alpine:3.6 echo -e 1\n22\n333\n\t4444\n55555\n666666\n\t7777777\n88888888\n999999999
1
22
333
        4444
55555
666666
        7777777
88888888
999999999

C:\Users\ctw>docker run --rm alpine:3.6 echo -e 1\n22\n333\n\t4444\n55555\n666666\n\t7777777\n88888888\n999999999
1
22
333
        4444
55555
666666
        7777777
88888888
999999999

C:\Users\ctw>docker run --rm alpine:3.6 echo -e 1\n22\n333\n\t4444\n55555\n666666\n\t7777777\n88888888\n999999999
1
22
333
        7777777
88888888
999999999

C:\Users\ctw>docker run --rm alpine:3.6 echo -e 1\n22\n333\n\t4444\n55555\n666666\n\t7777777\n88888888\n999999999
1       4444
55555
666666
        7777777
88888888
999999999
```

This behavior also occurs in run -it / exec -it.  If you use "docker run > tabs.txt" and then "type tabs.txt", Windows renders the output correctly.  I found identical behavior with centos:7.3.1611 and debian:jessie images.  `docker run --rm alpine:3.6 echo -e '1\n22\n333\n\t4444\n55555\n666666\n\t7777777\n88888888\n999999999'` exhibits correct behavior on all Docker-for-Linux platforms I tested.

Diagnostic ID: 7D920435-2951-4E4D-BBA1-87CCEDAAD0F0/2017-09-21_11-00-25

docker version:
```
Client:
 Version:      17.06.2-ce
 API version:  1.30
 Go version:   go1.8.3
 Git commit:   cec0b72
 Built:        Tue Sep  5 19:57:19 2017
 OS/Arch:      windows/amd64

Server:
 Version:      17.06.2-ce
 API version:  1.30 (minimum version 1.12)
 Go version:   go1.8.3
 Git commit:   cec0b72
 Built:        Tue Sep  5 19:59:19 2017
 OS/Arch:      linux/amd64
 Experimental: true
```

docker info:
```
Containers: 3
 Running: 0
 Paused: 0
 Stopped: 3
Images: 15
Server Version: 17.06.2-ce
Storage Driver: overlay2
 Backing Filesystem: extfs
 Supports d_type: true
 Native Overlay Diff: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host ipvlan macvlan null overlay
 Log: awslogs fluentd gcplogs gelf journald json-file logentries splunk syslog
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Init Binary: docker-init
containerd version: 6e23458c129b551d5c9871e5174f6b1b7f6d1170
runc version: 810190ceaa507aa2727d7ae6f4790c76ec150bd2
init version: 949e6fa
Security Options:
 seccomp
  Profile: default
Kernel Version: 4.9.41-moby
Operating System: Alpine Linux v3.5
OSType: linux
Architecture: x86_64
CPUs: 2
Total Memory: 1.941GiB
Name: moby
ID: 4SUM:MCU4:2LJK:CGHK:WEZ4:HXGQ:7I4C:VYAC:I24X:BTYZ:62CI:3FMJ
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Experimental: true
Insecure Registries: [redacted]
Live Restore Enabled: false
```

### Steps to reproduce the behavior

  1. Run 'cmd'
  2. docker run --rm alpine:3.6 echo -e 1\n22\n333\n\t4444\n55555\n666666\n\t7777777\n88888888\n999999999
  3. Re-run step 2 four or five times, until your terminal output reaches the bottom of the display window.

**Comment by ctwadv** [comment]
I am no longer observing this behavior.  This bug seems to be fixed as of 18.03.

Client:
 Version:       18.03.0-ce
 API version:   1.37
 Go version:    go1.9.4
 Git commit:    0520e24
 Built: Wed Mar 21 23:06:28 2018
 OS/Arch:       windows/amd64
 Experimental:  false
 Orchestrator:  swarm

Server:
 Engine:
  Version:      18.03.0-ce
  API version:  1.37 (minimum version 1.12)
  Go version:   go1.9.4
  Git commit:   0520e24
  Built:        Wed Mar 21 23:14:32 2018
  OS/Arch:      linux/amd64
  Experimental: false

Windows 10, Version 1709 (OS Build 16299.248)

**Comment by rn** [comment]
thanks for confirming. Closing
